src/centralserver/admin-ui/src/views/ManagementRequests/IconSvg.vue

```diff
@@ -0,0 +1,184 @@
+          fill="#211E1E"
+        />
+        <path
+          d="M13.1286 21.0637H5C4.99972 19.9608 5.25192 18.8725 5.73727 17.8822C6.22261 16.8918 6.92823 16.0257 7.80005 15.3502C8.67188 14.6747 9.68676 14.2078 10.7669 13.9852C11.5458 13.8246 12.3437 13.7942 13.1286 13.8931V21.0637Z"
+          fill="#211E1E"
+        />
+        <path
+          fill-rule="evenodd"
+          clip-rule="evenodd"
+          d="M15.7716 16.8692C15.6729 17.2842 15.6729 17.7166 15.7716 18.1316L15 18.5764L15.7778 19.9236L16.5493 19.4779C16.859 19.7712 17.2334 19.9876 17.6421 20.1094V21H19.1977V20.1094C19.6064 19.9876 19.9808 19.7712 20.2904 19.4779L21.062 19.9236L21.8398 18.5764L21.0682 18.1308C21.1667 17.716 21.1667 17.284 21.0682 16.8692L21.8398 16.4236L21.062 15.0764L20.2904 15.5221C19.9848 15.2328 19.6114 15.0134 19.1977 14.8906V14H17.6421V14.8906C17.2334 15.0124 16.859 15.2288 16.5493 15.5221L15.7778 15.0764L15 16.4236L15.7716 16.8692ZM17.8699 16.95C18.0158 16.8041 18.2136 16.7222 18.4199 16.7222C18.6261 16.7222 18.824 16.8041 18.9698 16.95C19.1157 17.0958 19.1976 17.2937 19.1976 17.4999C19.1976 17.7062 19.1157 17.9041 18.9698 18.0499C18.824 18.1958 18.6261 18.2777 18.4199 18.2777C18.2136 18.2777 18.0158 18.1958 17.8699 18.0499C17.724 17.9041 17.6421 17.7062 17.6421 17.4999C17.6421 17.2937 17.724 17.0958 17.8699 16.95Z"
+          fill="#211E1E"
+        />
+        <path
+          fill-rule="evenodd"
+          clip-rule="evenodd"
+          d="M15.7716 16.8692C15.6729 17.2842 15.6729 17.7166 15.7716 18.1316L15 18.5764L15.7778 19.9236L16.5493 19.4779C16.859 19.7712 17.2334 19.9876 17.6421 20.1094V21H19.1977V20.1094C19.6064 19.9876 19.9808 19.7712 20.2904 19.4779L21.062 19.9236L21.8398 18.5764L21.0682 18.1308C21.1667 17.716 21.1667 17.284 21.0682 16.8692L21.8398 16.4236L21.062 15.0764L20.2904 15.5221C19.9848 15.2328 19.6114 15.0134 19.1977 14.8906V14H17.6421V14.8906C17.2334 15.0124 16.859 15.2288 16.5493 15.5221L15.7778 15.0764L15 16.4236L15.7716 16.8692ZM17.8699 16.95C18.0158 16.8041 18.2136 16.7222 18.4199 16.7222C18.6261 16.7222 18.824 16.8041 18.9698 16.95C19.1157 17.0958 19.1976 17.2937 19.1976 17.4999C19.1976 17.7062 19.1157 17.9041 18.9698 18.0499C18.824 18.1958 18.6261 18.2777 18.4199 18.2777C18.2136 18.2777 18.0158 18.1958 17.8699 18.0499C17.724 17.9041 17.6421 17.7062 17.6421 17.4999C17.6421 17.2937 17.724 17.0958 17.8699 16.95Z"
+          fill="#663CDC"
+        />
+      </template>
+    </g>
+  </svg>
+</template>
+
+<script lang="ts">
+/**
+ * Component for SVG icons. Used for the few multi color icons because the icon font can not handle multiple colors.
+ * If more icons are added in the furure consider splitting the template (<g> tag parts) into
+ * separate vue components for each icon.
+ */
+import Vue from 'vue';
+
+export default Vue.extend({
+  name: 'IconSvg',
+  props: {
+    iconName: {
+      type: String,
+      default: 'box',
+    },
```

Comment:
Seems like it could be a good idea to validate the icon type here.

```suggestion
    iconName: {
      type: String,
      validator: (prop) =>
        [
          'register_client',
          'register_certificate',
          'delete_client',
          'delete_certificate',
          'change_owner',
        ].includes(prop),
      default: '',
    },
```

Reply:
Fixed